### DISCUSSION TITLE
creating your own epg

### DISCUSSION THREAD

**Discussion by ericek74** [K-vanc/Tempest-EPG-Generator#89] [Ideas]
Hi
thank you for the perfect plugin. it would be possible to set your epg for example youtube live channels, something like they have in wg++ default.ini, for example
* the shows for one day:
url_index {url|http://this-page-intentionally-left-blank.org/} * just an empty page
index_showsplit.scrub {single||||} * copies the html page
index_showsplit.modify {clear}
scope.range {(splitindex)|end}
index_variable_element.modify {addstart| 00:00-12:00 ## ROCK videoclips #### 12:00-23:59 ## ROCK videoclips ####} * 2 shows per day example

**Comment by K-vanc** [comment]
Hi, you can check Miscellaneous folder for dummy channel siteconfigs and below link for instructions with pictures;

https://github.com/K-vanc/Tempest-EPG-Generator/blob/main/Siteconfigs%2FMiscellaneous%2F0_info.md

You should pick suitable siteconfig for you and need to edit corresponding source file for your channel(s) shows. Hope it helps

**Comment by ericek74** [comment]
Super.It works great.Thank you.

**Reply by K-vanc** [reply]
You are welcome
